docs/content/getting-started/code-generation.md

```diff
@@ -329,23 +322,18 @@ You can find more examples in the
 
 ## What are Generators and Modules
```

Comment:
we may revisit this section later, when working on the `hof mod` documentation revamp